realtime/src/main/java/com/mx/path/api/lib/realtime/MemberCreateRequest.java

```diff
@@ -8,9 +8,13 @@
 import com.mx.common.accessors.UnauthorizedException;
 import com.mx.common.http.HttpStatus;
 import com.mx.common.http.MediaType;
-import com.mx.path.api.lib.realtime.models.MdxMember;
 import com.mx.path.gateway.net.Request;
+import com.mx.path.service.connection.realtime.model.MdxMember;
```

Comment:
This will be a breaking change. Could just duplicate the models and mark the old ones deprecated.

Reply:
I've added back the old models, deprecated them, fixed the usages (in the deprecated classes and deprecated tests)